packages/create-neutrino-project/commands/init/generator/utils.js

```diff
@@ -0,0 +1,28 @@
+module.exports.VERSIONS = {
+  NEUTRINO: '^7.2.1',
```

Comment:
So we definitely don't want to do this, which is what led us away from continuing with Neo. By hardcoding these versions here, we will be forced to release a new version of the project creator every time one of these needs a version bump. Instead, we can invoke the installation commands directly, and let the latest always be installed.